opwen_email_server/api/client_write.py

```diff
@@ -26,7 +26,7 @@ def __call__(self, client_id: str, upload_info: dict) -> Tuple[str, int]:
 
         resource_id = upload_info.get('resource_id')
 
-        tasks.written_store.delay(resource_id)
+        tasks.written_store.apply_async(args=resource_id, queue='written_store')
```

Comment:
Is it possible to configure this via Celery settings so that the user doesn't always have to remember to set the correct queue? E.g. via task routing? http://docs.celeryproject.org/en/latest/userguide/routing.html#automatic-routing

Reply:
Will switch over to task routing for this.